### PR TITLE
Update Android assets, separate GPS/BTE into individual packages

### PR DESCRIPTION
**Pull request by mattThousand**
Why:
Currently the assets and display names for both the GPS and BT
version of the android app reflect outdated designs from the original GPS app. We would like the display names to be Path Check GPS and Path Check BT respectively for the GPS
and BT product flavors, and for the assets to be specific to each product flavor.

This commit:
Separates the Android codebase into 2 packages containing separate resource assets (and also, importantly, putting all location tracking-related logic in the `gps` package). This commit also moves requests for the following permissions:
```
ACCESS_COARSE_LOCATION
ACCESS_BACKGROUND_LOCATION
ACCESS_FINE_LOCATION
```
into the `gps` package, and the following permissions
```
BLUETOOTH
BLUETOOTH_ADMIN
```
into the `bte` package.

Screen Shot 2020-05-27 at 1 45 06 PM

Screen Shot 2020-05-27 at 1 05 53 PM

### CODE REVIEW

File: android/app/src/gps/java/covidsafepaths/gps/MainApplication.java [hunk 1]
```diff
@@ -58,7 +56,7 @@ public void onCreate() {
     initializeFlipper(this); // Remove this line if you don't want Flipper enabled
     Realm.init(this);
     initializeGeolocationTransformer();
-    // Ignore assignment. Creating to begin heavy encryption work
+//    // Ignore assignment. Creating to begin heavy encryption work
```
Comment (tstirrat) [inline android/app/src/gps/java/covidsafepaths/gps/MainApplication.java]: weird comment change, revert

File: android/app/src/bte/res/values-sk/strings.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tstirrat) [inline android/app/src/bte/res/values-sk/strings.xml:1]: You will need to update `app/locales/pull.sh` and `push.sh` to use these new paths and pull into both bte/gps dirs.

**Review by tstirrat — CHANGES_REQUESTED** [review]

**Comment by tstirrat** [comment]
Also while you're here, the DetoxTest.java is in `src/androidTest/package/name/DetoxTest.java` but should have been in a java folder: `src/androidTest/java/package/name`

File: android/app/src/bte/java/covidsafepaths/bte/MainApplication.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  public ReactNativeHost getReactNativeHost() {
+    return mReactNativeHost;
+  }
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    MainApplication.context = getApplicationContext();
+    SoLoader.init(this, /* native exopackage */ false);
+    initializeFlipper(this); // Remove this line if you don't want Flipper enabled
```
Comment (johnschoeman) [inline android/app/src/bte/java/covidsafepaths/bte/MainApplication.java:49]: Do we want Flipper here? Im only vaguely aware of it as a debugging tool for RN? not sure if it something we are planning on using and its probably fine to leave in either way, but wondering if you have any context on what we are going to be using it for.

**Review by johnschoeman — DISMISSED** [review]
Overall the strategy looks good to me! I would just double check that the fastlane methods also dont need to be updated to accommodate these changes.

File: android/app/src/bte/java/covidsafepaths/bte/MainApplication.java (hunk 3 above)
Comment (mattThousand) [reply android/app/src/bte/java/covidsafepaths/bte/MainApplication.java:49]: Good question - I'd err on the side of leaving it in, assuming it's being used by other contributors.

**Comment by mattThousand** [comment]
@tstirrat if you have a moment, ready for another round of review 🙇‍♂️

**Review by johnschoeman — APPROVED** [review]
Continues to look good to me!

**Review by tstirrat — APPROVED** [review]

**Pull request merged**